app/models/spree/digital.rb

```diff
@@ -2,14 +2,19 @@ module Spree
   class Digital < ActiveRecord::Base
     belongs_to :variant
     has_many :digital_links, dependent: :destroy
+    has_many :drm_records, dependent: :nullify
 
     has_attached_file :attachment, path: ":rails_root/private/digitals/:id/:basename.:extension"
     do_not_validate_attachment_file_type :attachment
     validates_attachment_presence :attachment
 
     if Paperclip::Attachment.default_options[:storage] == :s3
       attachment_definitions[:attachment][:s3_permissions] = :private
-      attachment_definitions[:attachment][:s3_headers] = { :content_disposition => 'attachment' }
+      attachment_definitions[:attachment][:s3_headers] = { content_disposition: 'attachment' }
+    end
+
+    def create_drm_attachment(line_item)
```

Comment:
Why is it called `drm_attachment` here, but the model is named `drm_record`? I'd prefer a consistent naming.